test/lib/ufe/testAttribute.py

```diff
@@ -557,3 +557,50 @@ def testObservation(self):
         self.assertEqual(ball34Obs.notifications, 3)
         self.assertEqual(ball35Obs.notifications, 3)
         self.assertEqual(globalObs.notifications, 7)
+
+    # Run last to avoid file new disturbing other tests.
+    def testZAttrChangeRedoAfterPrimCreateRedo(self):
```

Comment:
Test-Driven Development: wrote test first, it failed (crashed), coded fix, test now passes.